src/engines/aurora/gui.cpp

```diff
@@ -70,8 +70,7 @@ void GUI::show() {
 			widget.show();
 	}
 
-	std::list<GUI*>::iterator iter;
-	for (iter = _childrenguis.begin(); iter != _childrenguis.end(); iter++)	{
+	for (std::list<GUI*>::iterator iter = _childGUIs.begin(); iter != _childGUIs.end(); iter++)	{
```

Comment:
Also, that `std::list<GUI*>` (here and everywhere else) should be `std::list<GUI *>`